Bottom: 287c062af6f6d11aa40c030e0fd7585040ee3c4b
Top:    b9b55189bbb06ca698f45303a5cf70bde63964ff
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Exec account script on exim4 container

Instead of starting a new container to add accounts,
execute script within existing container.


---
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index 43e565d4..21357723 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -127,5 +127,5 @@
 # Email accounts
 
 - name: populate email passwords
-  shell: 'docker run --rm --volumes-from {{ exim4_dv_name }} {{ exim4_image_name }}:{{ docker_image_tag }} add_account {{ item.value.user }} "{{ item.value.password }}"'
+  shell: 'docker exec {{ exim4_container_name }} /docker-entrypoint.sh add_account {{ item.value.user }} "{{ item.value.password }}"'
   with_dict: '{{ email_accounts }}'
